config/environments/test.rb

```diff
@@ -14,7 +14,7 @@
   # test suite. You never need to work with it otherwise. Remember that
   # your test database is "scratch space" for the test suite and is wiped
   # and recreated between test runs. Don't rely on the data there!
-  config.cache_classes = true
+  config.cache_classes = false
```

Comment:
Without turning this false, I couldn't run the test DB migrations `RAILS_ENV=test rails db:migrate`
Doesn't seem to affect performance or anything else so leaving it.